klasses/tasks.py

```diff
@@ -11,28 +11,29 @@
 log = logging.getLogger(__name__)
 
 
-def _cache_admissions(user_email, payable_klasses_json):
+def _cache_admissions(user_email, payable_klasses):
     """
     Caches the admissions for which the user is allowed to pay.
 
     Args:
         user_email (str): the email corresponding to an User
-        payable_klasses_json (dict): a dictionary built by the BootcampAdmissionClient._get_payable_klasses
+        payable_klasses (list): a list built by BootcampAdmissionClient._get_payable_klasses
     """
     try:
         user = User.objects.get(email=user_email)
     except User.DoesNotExist:
         log.exception('User with email %s does not exists', user_email)
         return
 
-    payable_klasses_keys = list(payable_klasses_json.keys())
+    payable_klasses_keys = [klass['klass_id'] for klass in payable_klasses]
```

Comment:
Take it or leave it: could be `payable_klasses_keys = payable_klasses_lookup.keys()` to lessen repetition.